src/components/ContactsView.js

```diff
@@ -1,12 +1,33 @@
 import { useEffect, useState } from "react"
 import { useParams } from "react-router-dom"
+import { useLocation } from "react-router-dom"
 
 function ContactsView() {
+
   const [contact, setContact] = useState(false)
 
+  const location = useLocation()
+  console.log("this is what is coming into location", location.state)
+
+
+
+
   //TODO: Get the contact to load from the params and fetch.
   //With useEffect, load the contact when params changes
   //and update contact state
+  useEffect(() => {
+    if (location.state) {
```

Comment:
good use of location.state something else to look at to do the same thing would be useParams, but good to see this working.